packages/edit-site/src/components/style-book/index.js

```diff
@@ -346,33 +348,60 @@ function StyleBook( {
 /**
  * Style Book Preview component renders the stylebook without the Editor dependency.
  *
- * @param {Object}   props            Component props.
- * @param {string}   props.path       Path to the selected block.
- * @param {Object}   props.userConfig User configuration.
- * @param {Function} props.isSelected Function to check if a block is selected.
- * @param {Function} props.onSelect   Function to select a block.
+ * @param {Object}  props            Component props.
+ * @param {Object}  props.userConfig User configuration.
+ * @param {boolean} props.isStatic   Whether the stylebook is static or clickable.
  * @return {Object} Style Book Preview component.
  */
-export const StyleBookPreview = ( {
-	path = '',
-	userConfig = {},
-	isSelected,
-	onSelect,
-} ) => {
+export const StyleBookPreview = ( { userConfig = {}, isStatic = false } ) => {
 	const siteEditorSettings = useSelect(
 		( select ) => select( siteEditorStore ).getSettings(),
 		[]
 	);
 	// Update block editor settings because useMultipleOriginColorsAndGradients fetch colours from there.
 	dispatch( blockEditorStore ).updateSettings( siteEditorSettings );
 
+	const [ section, onChangeSection ] = useSection();
+
+	const isSelected = ( blockName ) => {
```

Comment:
This is all beside the point, but I was looking into abstracting the stylebook component further so it can be dropped into the edit and view contexts without knowing too much about its own props.

https://github.com/WordPress/gutenberg/pull/67660/files#diff-ecd3155ef49b2da19da151cb9ac6813db5a887e4e7bd4ab1ca44b41ddba97b8b

I mean, this works too 😄

Reply:
I'm wondering if long term we'll still need the style book in the edit context at all. 

Reading through #66719, it seems the main reason global styles should stay in that right hand side sidebar in the editor is so folks can tweak styles while building their templates. While revisions should probably be accessible from wherever global styles can be edited, the style book is a more high-level tool so I'd question the usefulness of keeping it in that right hand sidebar.

Reply:
> I'm wondering if long term we'll still need the style book in the edit context at all.

👍🏻  I'd be for this if only to simplify the all the edit canvas view juggling etc we do.